.ci/test-r-package.sh

```diff
@@ -169,34 +169,34 @@ elif [[ $R_BUILD_TYPE == "cran" ]]; then
     if [[ "${TASK}" == "r-rchk" ]]; then
         echo "Checking R package with rchk"
         mkdir -p packages
-        cp ${PKG_TARBALL} packages
+        cp "${PKG_TARBALL}" packages
         RCHK_LOG_FILE="rchk-logs.txt"
-        docker run \
-            -v $(pwd)/packages:/rchk/packages \
+        { docker run \
+            -v "$(pwd)/packages:/rchk/packages" \
             kalibera/rchk:latest \
             "/rchk/packages/${PKG_TARBALL}" \
-        2>&1 > ${RCHK_LOG_FILE} \
+        > ${RCHK_LOG_FILE}; } 2>&1  \
```

Comment:
What specific shellcheck warning is the addition of these `{` / `}` attended to address? I feel that this makes the code quite difficult to read, and I'd like to understand what the purpose is.

Reply:
Same as above it's a SC2069 warning https://www.shellcheck.net/wiki/SC2206 it says it's needed to redirect both stdout and stderr streams to file

Reply:
Ok thank you. Please just move the `2>&1` to the end but revert the addition of `{` / `}`.